test/pycheck/temporary_table_test.py

```diff
@@ -0,0 +1,22 @@
+from .utils import Cursor, PG_MAJOR_VERSION, eprint
+
+
+def test_temporary_table_set_access_method(cur: Cursor):
+    if PG_MAJOR_VERSION >= 15:
```

Comment:
Can you use this instead to skip for lower PG versions?
```suggestion
@pytest.mark.skipif("PG_MAJOR_VERSION < 15", reason="SET ACCESS METHOD was introduced in PG15")
def test_temporary_table_set_access_method(cur: Cursor):
```

Reply:
Ok, this seems to be OK, this is my first time using python, thanks for the guidance.